client/my-sites/checkout/checkout/secure-payment-form.jsx

```diff
@@ -105,7 +105,7 @@ export class SecurePaymentForm extends Component {
 				// FIXME: The endpoint doesn't currently support transactions with no
 				//   payment info, so for now we rely on the credits payment method for
 				//   free carts.
-				newPayment = fullCreditsPayment;
+				newPayment = fullCreditsPayment();
```

Comment:
Wow, that's weird that that was just a `const`. Thanks for making it consistent.